app/styles/index.js

```diff
@@ -0,0 +1,13 @@
+var styles = {
```

Comment:
Good use of style sheet. As things get more complex and you build more components, you may consider putting styles for components in the component .js file. You can use this for global styles. Or you can even create new style sheets for every component. Traditional web development tells us to only have one style sheet as it's faster, but react builds down to one style sheet so you can make as many as you want when you're developing.